lldb/include/lldb/Interpreter/OptionValueSInt64.h

```diff
@@ -68,11 +68,10 @@ class OptionValueSInt64 : public Cloneable<OptionValueSInt64, OptionValue> {
   }
 
   bool SetDefaultValue(int64_t value) {
-    if (value >= m_min_value && value <= m_max_value) {
-      m_default_value = value;
-      return true;
-    }
-    return false;
+    assert(value >= m_min_value && value <= m_max_value &&
```

Comment:
A few things I'd like to make sure of:
- What happens in release configurations where assertions are compiled out? Might this allow you to set an invalid value?
- Can you call `SetDefaultValue` by typing a command in lldb? If so, an assertion seems like the wrong way of catching this. I ask because this is in OptionValue, which are involved in parsing commands in LLDB.

Reply:
No, the default value is set by LLDB itself. The situation this assert catches is someone defining a default value in the tablegen file that's larger or smaller than the mix/max value. AFAIK there is no way for a user to set a default value.

Currently, `SetMinValue` is only called for the terminal width and height. There are no other users.